community/knative/README.md

```diff
@@ -0,0 +1,200 @@
+# Knative
+
+This chart installs Knative components for Build, Serving, Eventing and Eventing-Sources.
+
+## Introduction
+
+- Visit [Knative Build](https://github.com/knative/build/blob/master/README.md) for more information about Build.
+- Visit [Knative Eventing](https://github.com/knative/eventing/blob/master/README.md) for more information about Eventing.
+- Visit [Knative Eventing Sources](https://github.com/knative/eventing-sources/blob/master/README.md) for more information about Eventing Sources.
+- Visit [Knative Serving](https://github.com/knative/serving/blob/master/README.md) for more information about Serving.
+
+## Chart Details
```

Comment:
Question: Should we add the recommended chart details back here, seen as we only have one readme for icp